src/components/atoms/OcDrop/OcDrop.vue

```diff
@@ -206,6 +202,7 @@ export default {
 }
 
 .oc-drop {
+  box-shadow: 0 5px 15px rgba(0,0,0,.08);
```

Comment:
You could extend `.oc-box-shadow-medium` here, right?

Reply:
Yeah true, I'll add the class directly to the DOM element